sdk/async_api/spot_trade_bust.py

```
@@ -0,0 +1,48 @@
+    additional_properties = getattr(self, "additional_properties")
+    if additional_properties is not None:
+      for key, value in additional_properties.items():
+        # Never overwrite existing values, to avoid clashes
+        if not key in serialized_self:
+          serialized_self[key] = value
+
+    return serialized_self
+
+  @model_validator(mode='before')
+  @classmethod
+  def unwrap_additional_properties(cls, data):
```

Ruff (0.15.5)

[warning] Missing return type annotation for classmethod `unwrap_additional_properties`

(ANN206)